deployment/local-jvm/bash-scripts/verify.sh

```diff
@@ -10,11 +10,11 @@ if [ -d $DIR ]; then
   if [ -f $FILE ]; then
     num=$(wc -l $FILE | awk '{ print $1 }')
 
-    #check the length of the exampleOutput text file and pass if it's 944
-    if [ $num == 944 ]; then
-      echo "Success - Number of lines was 944"
+    #check the length of the exampleOutput text file and pass if it's 781
+    if [ $num == 781 ]; then
+      echo "Success - Number of lines was 781"
     else
```

Comment:
This feels really brittle. Any minor tweaks and this will potentially break - as we see here with the change from 944 to 781 lines. I think this should be refactored into "CONTAINS" and "DOES NOT CONTAIN" rather than a specific number of lines.

Reply:
PAL-544 on the backlog will tackle this issue, the number of lines check was only ever a stop gap between tickets